.gitignore

```diff
@@ -1,8 +1,12 @@
 /node_modules/
 
+/example/node_modules/
```

Comment:
I don't think you need these two, it should work recursively. Unless you have some IDE that would have a problem with it? In that case, I would rather create separate `.gitignore` in an `example` folder.

Reply:
the leading leading slash only matches root of the project, so just the top level node_modules

Reply:
Alright, missed that one, let's change it to `node_modules` then, I don't see a reason why not.